shared/actions/chat2/index.tsx

```diff
@@ -2557,7 +2557,7 @@ const mobileNavigateOnSelect = (
     logger.info(
       `mobileNavigateOnSelect: navigating to selected: selected: ${state.chat2.selectedConversation} param: ${conversationIDKey}`
```

Comment:
Change the log line to reflect what it is actually doing.

Reply:
Isn't it already doing that